.travis.yml

```diff
@@ -0,0 +1,36 @@
+sudo: false
```

Comment:
```suggestion
```

This key is officially obsolete and ignored. I just learned this today on another project, but they've been changing up how their infrastructure works.